### ISSUE TITLE
RETURN QUERY EXECUTE should not warn about unmodified OUT variables

### ISSUE THREAD

**Issue by brankoradovanovic-mcom** [okbob/plpgsql_check#199]
A function e.g. like this:

```
CREATE OR REPLACE FUNCTION test()
 RETURNS TABLE(id integer, message text)
 LANGUAGE plpgsql
AS $function$
declare
	v_sql text;
begin
	select 'values ' || string_agg(format('(%s, %L)', n, 'Test message'), ', ' order by n)
	into v_sql
	from generate_series(1, 10) s(n);

	return query execute v_sql;
end
$function$;
```

...will trigger the following warnings:

```
1	warning extra:00000:OUT variable "id" is maybe unmodified
2	Detail: cannot to determine result of dynamic SQL
3	warning extra:00000:OUT variable "message" is maybe unmodified
4	Detail: cannot to determine result of dynamic SQL
```

This is a false positive because:
1. The shape of the result set is already declared in the function’s `RETURNS TABLE (...)`.
2. Inside the function the OUT columns are never referenced, so there is nothing to “forget”.

There is apparently no pragma to silence this specific warning other than `PERFORM plpgsql_check_pragma('disable:check');`

Since the code in the above example is "correct by definition", I would argue that `return query execute` should never trigger warnings if it's used with dynamic SQL.

(Nitpick: "cannot to determine" -> "cannot determine")
